From 7ffb9cc0def0f36205fd882858fd1728340b85be Mon Sep 17 00:00:00 2001
From: Liu Xiaowen <b37945@freescale.com>
Date: Thu, 26 Sep 2013 17:34:46 +0800
Subject: [PATCH 473/635] ENGR00215174 enlarge input period_size to fix buffer
 overflow on WFD.

enlarge input period_size to fix buffer overflow on WFD.
because AudioFlinger get min buffer size from the primary
input device to calculate total buffer size.

Signed-off-by: Liu Xiaowen <b37945@freescale.com>
---
 alsa/tinyalsa_hal.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/alsa/tinyalsa_hal.c b/alsa/tinyalsa_hal.c
index d16a55a..cb57878 100644
--- a/alsa/tinyalsa_hal.c
+++ b/alsa/tinyalsa_hal.c
@@ -71,9 +71,9 @@
 /* number of periods for low power playback */
 #define PLAYBACK_LONG_PERIOD_COUNT  8
 /* number of periods for capture */
-#define CAPTURE_PERIOD_SIZE  512
+#define CAPTURE_PERIOD_SIZE  1024
 /* number of periods for capture */
-#define CAPTURE_PERIOD_COUNT 8
+#define CAPTURE_PERIOD_COUNT 4
 /* minimum sleep time in out_write() when write threshold is not reached */
 #define MIN_WRITE_SLEEP_US 5000
 
-- 
1.8.0

